src/test/java/LoginTests.java

```diff
@@ -41,29 +42,29 @@ public static void LoginInvalidEmailValidPasswordTest () {
 
     }
 
-    @Test(enabled = true, priority = 3, description = "LoginValidEmailValidPasswordTest")
-    public static void updateProfileNameTest () throws InterruptedException {
-
-        navigateToPage();
-
-        provideEmail("demo@class.com");
-        providePassword("te$t$tudent");
-        clickSubmit();
-
-        Thread.sleep(2000);
-        clickAvatarIcon();
-
-        String randomName = generateRandomName();
-
-        provideCurrentPassword("te$t$tudent");
-        provideProfileName(randomName);
-        clickSaveButton();
-
-        Thread.sleep(2000);
-        WebElement actualProfileName = driver.findElement(By.cssSelector("a.view-profile>span"));
-        Assert.assertEquals(actualProfileName.getText(), randomName);
-
-    }
+//    @Test(enabled = true, priority = 3, description = "LoginValidEmailValidPasswordTest")
+//    public static void updateProfileNameTest () throws InterruptedException {
+//
+//        navigateToPage();
+//
+//        provideEmail("demo@class.com");
+//        providePassword("te$t$tudent");
+//        clickSubmit();
+//
+//        Thread.sleep(2000);
+//        clickAvatarIcon();
+//
+//        String randomName = generateRandomName();
+//
+//        provideCurrentPassword("te$t$tudent");
+//        provideProfileName(randomName);
+//        clickSaveButton();
+//
+//        Thread.sleep(2000);
+//        WebElement actualProfileName = driver.findElement(By.cssSelector("a.view-profile>span"));
+//        Assert.assertEquals(actualProfileName.getText(), randomName);
+//
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.